src/injected/injected.ts

```diff
@@ -2,13 +2,12 @@ import {createDeferredPromise} from '../util/DeferredPromise.js';
 import * as Poller from './Poller.js';
 import * as util from './util.js';
 
-Object.assign(
-  self,
-  Object.freeze({
-    InjectedUtil: {
-      ...Poller,
-      ...util,
-      createDeferredPromise,
-    },
-  })
-);
+const PuppeteerUtil = Object.freeze({
+  ...Poller,
+  ...util,
+  createDeferredPromise,
```

Comment:
let's not include stuff that is not used before we need it. I believe only createDeferredPromise is used?

Reply:
I've kept `util` since we use it for testing.